src/hammer-tech/hammer_tech.py

```diff
@@ -212,17 +213,24 @@ def __init__(self):
         self.config = None  # type: TechJSON
 
     @classmethod
-    def load_from_dir(cls, technology_name: str, path: str) -> "HammerTechnology":
-        """Load a technology from a given folder.
+    def load_from_dir(cls, technology_name: str, paths: List[str]) -> Optional["HammerTechnology"]:
+        """Load a technology from a given set of folders.
 
         :param technology_name: Technology name (e.g. "saed32")
-        :param path: Path to the technology folder (e.g. foo/bar/technology/saed32)
+        :param paths: A list of paths to the technology folders (e.g. foo/bar/technology/saed32)
         """
-
-        with open(os.path.join(path, "{technology_name}.tech.json".format(technology_name=technology_name))) as f:
-            json_str = f.read()
-
-        return HammerTechnology.load_from_json(technology_name, json_str, path)
+        for path in paths:
```

Comment:
I'm not sure I would put this `paths` for loop in here. It might be related to the unit test failures. Can take a look at some point.

Reply:
This passed before its probably something silly.

Reply:
Yeah I think some technology prefixing is inconsistent somewhere. I do think `load_from_dir` makes slightly more sense if it loads a single dir though, and instead we let the driver load from multiple dirs (this is consistent with tool plugin loading)

Reply:
Yeah I agree. My original load_from_dir change was keeping the name accurate, but in order to do the yaml/json detection in there I had to move the paths loop, or duplicate code/work.